### PR TITLE
Fix audit to be happier

### PR DESCRIPTION
**Pull request by rbtcollins**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -71,7 +72,7 @@ version = "1.0"
 
 [dev-dependencies]
 # async-std = "1.10.0"
-criterion = "0.3.5"
+criterion = "0.3.5"     # Remember to update audit.toml
```
Comment (sighphyre) [inline Cargo.toml:75]: I'm not 100% sure what this comment means. Does this reference the change that's in this PR or is this a reminder to a future maintainer that they need to do something important? If it's the second, is there a link with more details that we can leave for future persons?

Comment (rbtcollins) [reply Cargo.toml:75]: the audit failure that is whitelisted is brought into the dependency tree via criterion. When we get a dependabot PR updating criterion we need to go and update audit.toml.

Comment (sighphyre) [reply Cargo.toml:75]: Ah! Yes, that makes sense!

**Review by sighphyre — APPROVED** [review]
LGTM!

**Pull request merged**